wiki/Help_Centre/en.md

```diff
@@ -626,7 +628,7 @@ Once all of the `.osz` files within the pack are in the Songs directory, hitting
 
 **The vast majority of the time, this is due to a high number of tickets coming into our system at once.**
 
-You can always send a single reply to your currently-open ticket asking about the state of your case, but make sure not to do this more than once or twice a week.
+You can always send a single reply to your currently-open ticket asking about the state of your case, but make sure not to do this more than once or twice a week. You can also contact us on twitter at @osusupport to ask about your ticket.
```

Comment:
`currently-open` -> `currently open`, no?

Reply:
currently-open is technically correct (and would actually be better in this instance) but i'll remove the hyphen to make it simpler

Reply:
most style guides recommend using hyphens for compound adjectives, *except* where the first word is the adverb "very" or an adverb ending in "ly"

so `currently open` is technically correct too =) one of the many dumb gotcha's of english